src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html

```diff
@@ -7,19 +7,14 @@
 >
     <div class="m-2">
         <div *ngIf="!viewOnly; else readOnlyFeedback" class="row align-items-start" (drop)="updateFeedbackOnDrop($event)" (dragover)="$event.preventDefault()">
-            <div class="form-group col-8" [style.max-width]="'500px'">
+            <div class="form-group col-8">
                 <label jhiTranslate="artemisApp.assessment.detail.feedback"></label>
                 <textarea class="form-control" rows="5" [(ngModel)]="feedback.detailText" [readOnly]="viewOnly"></textarea>
             </div>
-            <div class="form-group col-1" [style.max-width]="'100px'">
+            <div class="form-group col-2">
                 <label jhiTranslate="artemisApp.exercise.score"></label>
                 <input class="form-control" type="number" step="0.5" [(ngModel)]="feedback.credits" [readOnly]="viewOnly || disableEditScore" />
```

Comment:
The padding of the text is pretty high, adding the class [px-1](https://getbootstrap.com/docs/5.0/utilities/spacing/) would reduce that and even allow for 5 digits.
Currently:
![grafik](https://user-images.githubusercontent.com/61357727/123420034-46e5d480-d5bb-11eb-86c4-c3740abe744b.png)
With px-1
![grafik](https://user-images.githubusercontent.com/61357727/123419987-36355e80-d5bb-11eb-8298-f7c066fd8c78.png)
(Only tried it out through the browser editor, needs to be tested properly)

Reply:
this is the text exercise feedback not programming one, but we could change in this PR dunno....

Reply:
Oh I'm sorry, I just assumed it would be the same code for all of them. My bad

Reply:
Checked again, the same fix could also be applied to programming exercises:
Current:
![grafik](https://user-images.githubusercontent.com/61357727/123438336-e06bb100-d5d0-11eb-9076-4063f5d53150.png)
With px-1:
![grafik](https://user-images.githubusercontent.com/61357727/123438382-ecf00980-d5d0-11eb-9905-e6a2eeef7f0d.png)
(Both time 100 points)